components/JEDI/AllyshipSpaceCard.js

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+
+import styles from '../../styles/pages/jedi.module.scss';
+
+// TODO: flesh out docs
+/**
+ * creates an allyship space card from input parameters, using
+ * next/image and next/link
+ */
+function AllyshipSpaceCard({ title, date, location, description, pic, alt, rsvp, slides }) {
```

Comment:
IIRC we have a component that's similar to this created for the techgala page called `ProjectCard`.

I know these two components are functionally different, but thoughts on if they're similar enough to warrant creating an "EventCard" component (like a super class of these two). Might be a good thing to do in preparation for the events page, which I imagine will leverage a similar layout as these.

Reply:
I see what you're saying, though the styles are quite different on each card. On my end, I'm happy to keep them separate, but also interested to explore creating a generic Card that has a few configurable options (in this case, which side the image goes on, etc.). Just worried that then we'll get way too much required customization. What do you think?

Reply:
I think what we can do here is leverage the `children` prop to get the customization we need. Then for the super class we'd just need to specify and image and where the text should flow (left, right, or below).

The hope is that we can have some generic css that will format the height and width of the children (usually just text) to flow with the image.

Reply:
Sounds good to me, let's see if I can get that in relatively quickly for this change.